flake.nix

```diff
@@ -0,0 +1,59 @@
+{
+  description = ''
+    Flake for the cf-cli-plugin for app-autoscaler
+
+    For more on app-autoscaler, see: <https://github.com/cloudfoundry/app-autoscaler-release>
```

Comment:
Why is the link surrounded by `<>`?

Reply:
Descriptions are – as far as I remember – supposed to be markdown-formatted. Those surroundings are defined here: <https://spec.commonmark.org/0.31.2/#link-destination>

Some static-site generators require them sometimes so that the URL gets converted into an actual clickable link. But in general having some quoting for links is more stable (i.e. it allows URLs to work that otherwise would not.)

Reply:
Alright, thanks for the info : )